garnett/dcdfilereader.py

```diff
@@ -244,6 +242,8 @@ def load_arrays(self):
                            dtype=TYPE_PROPERTIES['types'])
         typeid = np.asarray([f._typeid for f in self.frames],
                             dtype=PARTICLE_PROPERTIES['typeid'])
+        box = np.asarray([f._box for f in self.frames],
+                         dtype=FRAME_PROPERTIES["box"])
```

Comment:
There is not an official PEP8 recommendation but most people prefer single quotes for dict keys, and it matches what's above.
```suggestion
                         dtype=FRAME_PROPERTIES['box'])
```